README.md

```diff
@@ -0,0 +1,191 @@
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🎬 Representation](#representation)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hotel Hub <a id="about-project"></a>
+
+**Hotel Hub** is a website that offers hotel booking and reservation services for travelers, providing a convenient way to find and book accommodations. You can select available rooms for a graphic list on the home page of the website, or you can just click on the reserve menu item and see the full list of available rooms. You can visit the repository for the [backend](https://github.com/ProgramKingAbel/hotel-hub-backend.git) of this project.
+
+## 🛠 Built With <a id="built-with"></a>
+
+### Tech Stack <a id="tech-stack"></a>
+
+<details>
+  <summary>React</summary>
+  <ul>
+    <li><a>https://rubyonrails.org/</a></li>
+    <li><a>https://react.dev/</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a id="key-features"></a>
+
+- **Ability to see the featured rooms on the homepage**
+- **The ability to reserve rooms based on selecting the date and room name**
+- **The ability to add rooms to the database**
+- **The ability to delete rooms**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a id="live-demo"></a>
+
+Live demo for the website [Coming Soon]().
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a id="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project, you need to:
+
+- Install React
+- Install React Router Dom
+- Install Redux
+- Install Redux toolkit
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+``` 
+  git clone https://github.com/ProgramKingAbel/hotel-hub-frontend.git 
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+ 
+ ```cd hotel-hub-frontend
+    npm install
+ ```
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+
+To run the project execute the following command:
+
+``` 
+  npm start
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run Tests <a id="run-tests"></a>
+
+To run the tests, execute the following command:
+```
+npm test
```

Comment:
- [ ] When I run the tests, this happens:

![image](https://github.com/ProgramKingAbel/hotel-hub-frontend/assets/57421823/a8b66c70-6b2d-4b56-8e5d-b42e0ce4c56e)

tip" It seems this line is giving us trouble:

![image](https://github.com/ProgramKingAbel/hotel-hub-frontend/assets/57421823/0b450638-0462-4442-bfb3-bd7c169e084e)

I deleted it and the tests worked.